code/game/machinery/suit_storage_unit.dm

```diff
@@ -278,8 +280,10 @@
 				close_machine()
 		if ("disinfect")
 			if (occupant && safeties)
+				say("Alert: safeties triggered, occupant detected!")
```

Comment:
does this show up in runechat?

Reply:
Yup, the joys of Say; it does a to-chat AND a runechat message